src/components/AutocompleteSelectMenu/AutocompleteSelectMenu.tsx

```diff
@@ -80,7 +80,21 @@ const AutocompleteSelectMenu: React.FC<AutocompleteSelectMenuProps> = props => {
   } = props;
   const classes = useStyles(props);
 
-  const [inputValue, setInputValue] = React.useState(displayValue || "");
+  const [isOnBlur, setIsOnblur] = React.useState(false);
+
+  const getDefaultCase = () => {
+    if (!!isOnBlur && displayValue === undefined) {
+      return null;
+    }
+    if (!!isOnBlur && !!displayValue) {
+      return "";
+    }
+  };
+
+  const [inputValue, setInputValue] = React.useState(
+    displayValue || getDefaultCase()
+  );
```

Comment:
“_!!isOnBlur && displayValue === undefined will be always false because double negation (why do you double negate boolean?)_” - typo, it should be `||` there.  It turned out that `undefined` also satisfies on of the conditions that we need. So the solution, as I tested it, fixes this bug event with this typo. 
Double bang `!! ` actually works as coercion to a Boolean value. It’s not double Boolean negation. Yes, we don’t need that coercion but we do the other one.

“_Same goes for !!isOnBlur && !!displayValue as !!isOnBlur is false too._ “ – you can have `true` and `true` which equals to `true`.

“_As an effect this function returns nothing_” - this function returns `undefined`  or `empty string`.